pom.xml

```diff
@@ -221,6 +223,13 @@
         <directory>src/main/nsis/bin</directory>
         <filtering>true</filtering>
       </resource>
+      <resource>
+        <directory>${DLLFolder}</directory>
+        <filtering>false</filtering>
```

Comment:
What does this filtering=false do?

Reply:
From https://books.sonatype.com/mvnref-book/reference/resource-filtering-sect-description.html:

Using Maven resource filtering you can reference Maven properties and then use Maven profiles to define different configuration values for different target deployment environments. 

Given that we are not referencing any Maven properties for including DLLs for Windows environment, I set <filtering> to false.